backend/api/Mapping/Converters/ParcelConverter.cs

```diff
@@ -15,5 +18,10 @@ public static int ConvertPID(string pid)
             int.TryParse(pid?.Replace("-", "") ?? "", out int value);
             return value;
         }
+
+        public static string ConvertPIDToDash(string pid)
+        {
+            return String.Join("-", Regex.Split(pid.Replace("-", "").PadLeft(9, '0'), @"(?<=\G.{3})(?!$)"));
```

Comment:
add some fault tolerance to the backend for pids in an invalid format.

Reply:
Add some documentation for the function.

Reply:
Added in follow-up pr.